src/components/Home/Aditya/Week3/components/newUser/NewUser.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import { NewUserDiv } from './newUserStyles';
+import { removeWhitespace } from '../../utils/utils';
+
+interface User {
+  name: string;
+  status: string;
+}
+
+const NewUser: React.FC = () => {
+  const [users, setUsers] = useState<User[]>(JSON.parse(localStorage.getItem('users') ?? '[]'));
+  const [isValidUsername, setIsValidUsername] = useState<boolean>(false);
+  const [newUser, setNewUser] = useState<User>({
+    name: '',
+    status: ''
+  });
+
+  const statusChoices: string[] = ['Busy', 'Active', 'Do not disturb', 'Idle'];
```

Comment:
To reduce the code redundancy you can use export statusOption function which keshav made in his folder so that repetition of code will not be there